extensions/oidc/runtime/src/test/java/io/quarkus/oidc/runtime/KnownOidcProvidersTest.java

```diff
@@ -513,7 +533,8 @@ public void testOverrideTwitchProperties() throws Exception {
         tenant.credentials.clientSecret.setMethod(Method.BASIC);
         tenant.authentication.setForceRedirectHttpsScheme(false);
 
-        OidcTenantConfig config = OidcUtils.mergeTenantConfig(tenant, KnownOidcProviders.provider(Provider.FACEBOOK));
+        tenant.setProvider(Provider.TWITCH);
```

Comment:
This is intentional switch from `FACEBOOK` to `TWITCH`, I think the previous state was a typo.